aten/src/ATen/cuda/CublasHandlePool.cpp

```diff
@@ -0,0 +1,42 @@
+#include <ATen/cuda/CUDAContext.h>
+#include <ATen/cuda/detail/DeviceThreadHandles.h>
+
+namespace at { namespace cuda {
+namespace {
+
+void createCublasHandle(cublasHandle_t *handle) {
+  TORCH_CUDABLAS_CHECK(cublasCreate(handle));
+}
+
+void destroyCublasHandle(cublasHandle_t handle) {
+// this is because of something dumb in the ordering of
+// destruction. Sometimes atexit, the cuda context (or something)
+// would already be destroyed by the time this gets destroyed. It
+// happens in fbcode setting. @colesbury and @soumith decided to not destroy
```

Comment:
Do we need to do the same thing for cuBlas handle? @colesbury @soumith

Reply:
probably, yes.